packages/language-server/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can we use lowercase filenames?

Reply:
Sure, but I would rather focus in that later. I prefer consistency (`readme.md`, `license`, `changelog.md`) over changing this for the newly added files here. I need to dive into Changesets to figure out how this affects changelog generation.

Reply:
I’m marking related comments as resolved.